db/migrate/20230119074426_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ]  according to project requirement kindly reference user with author i.e `t.references :author, foreign_key: { to_table: 'users' } `